metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -65,6 +65,17 @@
 BOOTSTRAP_MCES_URL = f"{DOCKER_COMPOSE_BASE}/{BOOTSTRAP_MCES_FILE}"
 
 
+def download_sample_data() -> str:
```

Comment:
ideally this method should live in the sample_source.py file

generally for code organization, cli can depend on sources, but sources should not depend on the CLI

Reply:
moved